MaloWBotLK/Functions.lua

```diff
@@ -292,6 +296,59 @@ function mb_getMySpecName()
 	return name
 end
 
+-- Scans through all bag slots looking for item by name
+function mb_GetItemLocation(itemName)
+	for bag = 0, 4 do
+		for slot = 1, GetContainerNumSlots(bag) do
+			local texture = GetContainerItemInfo(bag, slot)
```

Comment:
Here I would probably do local _, _, _, _, _, ... itemLink = GetContainerItemInfo
That way you can nil-check the itemLink instead on the row below, and you already have the itemLink ready for the row below that